lib/PostEditorBlocks/MarketingOptinBlock.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
In the new files we add the statement `declare(strict_types=1);` Could you please add it here as well? And add types to all methods in this class?